pkg/payload/render.go

```diff
@@ -90,10 +107,40 @@ func renderDir(renderConfig manifestRenderConfig, idir, odir string, skipFiles s
 			continue
 		}
 
-		rraw, err := renderManifest(renderConfig, iraw)
-		if err != nil {
-			errs = append(errs, err)
-			continue
+		var rraw []byte
+		if processTemplate {
+			rraw, err = renderManifest(renderConfig, iraw)
+			if err != nil {
+				errs = append(errs, fmt.Errorf("render template %s from %s: %w", file.Name(), idir, err))
+				continue
+			}
+		} else {
+			rraw = iraw
+		}
+
+		if len(filterGroupKind) > 0 {
+			manifests, err := manifest.ParseManifests(bytes.NewReader(rraw))
+			if err != nil {
+				errs = append(errs, err)
```

Comment:
Thanks!  I've pushed fd14fa1f -> 455abd6a with that, except I dropped `failed to`, because I expect the `err` to already be sufficiently clear that something failed.